test/unit/cinstance_test.rb

```diff
@@ -706,26 +711,23 @@ class DescriptionValidationTest < ValidationsTest
       assert @cinstance.valid?
     end
 
-    test 'provider has multi apps enabled: require description if a human interaction is happening' do
+    test 'provider has multi apps enabled: not require description if a human interaction is happening' do
       @provider.settings.allow_multiple_applications!
       @provider.settings.show_multiple_applications!
       @cinstance.validate_human_edition!
-
-      assert @cinstance.invalid?
-      assert @cinstance.errors[:description].presence
+      assert @cinstance.valid?
     end
 
     test 'service requires intentions: not require description' do
       @service.update(intentions_required: true)
       assert @cinstance.valid?
     end
 
-    test 'service requires intentions: require description if a human interaction is happening' do
+    test 'service requires intentions: not require description if a human interaction is happening' do
       @service.update(intentions_required: true)
       @cinstance.validate_human_edition!
 
-      assert @cinstance.invalid?
-      assert @cinstance.errors[:description].presence
+      assert @cinstance.valid?
```

Comment:
```suggestion
      assert @cinstance.valid?
      assert_nil @cinstance.errors[:description].presence
```

Reply:
Hm... isn't that redundant @josemigallas ?
I think by definition if `.valid?` returns true, it means that the `.errors` is empty.